GitExtensions/Project.Publish.targets

```diff
@@ -249,7 +249,8 @@
 
       <!-- This property comes from GitInfo package, but can be overriden on AppVeyor -->
       <_PublishPortableCommitHashSuffix Condition="'$(GitCommit)' != ''">-$(GitCommit)</_PublishPortableCommitHashSuffix>
-      <_PublishPortableFileName>GitExtensions-Portable$(_PublishPortableVersionSuffix)$(_PublishPortableCommitHashSuffix).zip</_PublishPortableFileName>
+      <_PublishPortableFileName Condition="'$(TargetPlatform)' == 'x86'">GitExtensions-Portable$(_PublishPortableVersionSuffix)$(_PublishPortableCommitHashSuffix).zip</_PublishPortableFileName>
+      <_PublishPortableFileName Condition="'$(TargetPlatform)' != 'x86'">GitExtensions-Portable-$(TargetPlatform)$(_PublishPortableVersionSuffix)$(_PublishPortableCommitHashSuffix).zip</_PublishPortableFileName>
```

Comment:
I think we should always add the platform to the name to avoid confusion.

Reply:
Yes, I agree with this explicit platform in the build artifacts' names.